Gemfile.rails42

```diff
@@ -1,12 +1,3 @@
-source 'https://rubygems.org'
-gemspec
-
-gem 'jruby-openssl', :platforms => :jruby
-gem 'rubocop', '~> 0.57.2', require: false
-
-group :test, :remote_test do
-  # gateway-specific dependencies, keeping these gems out of the gemspec
-  gem 'braintree', '>= 2.50.0'
-end
+eval File.read('Gemfile')
```

Comment:
FWIW Bundler has `eval_gemfile`, eg. https://github.com/search?q=eval_gemfile&type=Code

Reply:
Do you know if there's an advantage? I just looked at its implementation, and there's nothing obvious (it handles certain errors a bit better, after a fashion, but it's not as if a stack trace saying the file couldn't be found is gonna be hard to parse). I'd definitely have done this if I'd known about it; just wondering if it actually makes a difference.

Reply:
Not much, mostly cosmetic and reflects this is a common pattern.

Reply:
Also allowed shoving the other `Gemfile`s in their own directory. Thank you for the suggestion, @varyonic ; landed in dde5ab42d5c9